src/main/java/com/minecolonies/coremod/colony/Colony.java

```diff
@@ -1977,4 +1989,16 @@ public int getLastContactInHours()
     {
         return lastContactInHours;
     }
+
+    @Override
+    public boolean hasWillRaidTonight()
+    {
+        return willRaidTonight;
+    }
+
+    public void setWillRaidTonight(final Boolean willRaid)
+    {
+        System.out.println("Debug5");
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Replace this use of System.out or System.err by a logger. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonar.minecolonies.com/coding_rules#rule_key=squid%3AS106)